src/index.html

```diff
@@ -11,7 +11,141 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="field-set">
+        <legend>Personal information</legend>
+        <label class="input">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            min="1"
+            max="100"
+            name="Old"
+          >
+        </label>
+        <label class="input">
+          Full date of birth:
+          <input type="date" value="mm/dd/yyyy">
+        </label>
+        <label class="input">
+          I accept the term of the agreement
+          <input type="checkbox" name="Agreement">
+        </label>
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>Registration</legend>
+        <label class="input">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            required
+            name="email"
+          >
+        </label>
+        <label class="input">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="24"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>An interesting fact about you!</legend>
+        <label class="input">
```

Comment:
remove this label
when I click on `Do you love cats?`, it chooses the first option
<img width="228" alt="image" src="https://user-images.githubusercontent.com/102411130/229436058-4371d171-f7ac-4a86-8323-e55ed6392313.png">

Reply:
leave only for inputs here